connexion/utils.py

```diff
@@ -205,3 +206,40 @@ def iscorofunc(func):
     else:  # pragma: 3 no cover
         # there's no asyncio in python 2
         return False
+
+
+def yamldumper(openapi):
+    """
+    Returns a nicely-formatted yaml spec.
+    :param openapi: a spec dictionary.
+    :return: a nicely-formatted, serialized yaml spec.
+    """
+    def should_use_block(value):
```

Comment:
can you add a comment here?
I have no idea what this sequence of unicode characters is.